src/components/AppShell.astro

```diff
@@ -0,0 +1,15 @@
+---
+import Header from './Header.vue';
+import DesktopNavigation from './DesktopNavigation.astro';
+const { path, navigation } = Astro.props;
+
+const navPath = path.endsWith('/') ? path.slice(0, -1) : path;
+---
+
+<div class="flex h-screen w-screen">
+  <Header client:load path={path} navigation={navigation} />
```

Comment:
@alexBinns should the the path here be `navPath`?

Reply:
Yup